docs/source/sections/install.rst

```diff
@@ -0,0 +1,23 @@
+.. _2020_virtual_intro:install:
+
+*******************************************
+AiiDA installation
+*******************************************
+
+To install AiiDA follows the instructions on:
+
+   `https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/get_started.html`_
+
+.. _https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/get_started.html: https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/get_started.html
+
+Then, install the `aiida-quantumespresso` plugin:
+
+   `https://github.com/aiidateam/aiida-quantumespresso`_
+
+.. _https://github.com/aiidateam/aiida-quantumespresso: https://github.com/aiidateam/aiida-quantumespresso
+
+Now, after installing and compiling quantum ESPRESSO, you can set it up on AiiDA following the tutorial instructions.
+
+.. important::
+
+   Please download **today** any file you may want to keep. Tomorrow, the virtual machines will be deleted and, the user files will be lost.
```

Comment:
I think we can leave the cluster up till the end of the week. We did this for the BIGMAP tutorial as well.

Reply:
Hi @mbercx, I got all your suggestions and added to a new branch. I will create a new PR and close this one.